linAssgn1

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+clear
+echo -n "Enter the Process Id to be killed : " 
+read prid
+if [ $prid -le 1 -o $prid -ge 32768 ]
```

Comment:
So ideally you would read the max pid at run time by reading this file using cat. Can you do this?